doc/Command-Reference.md

```diff
@@ -9557,20 +9557,20 @@ Another commonly deployed blackhole avoidance feature: dynamic route priority ad
 to avoid large routes churn during BGP restart.
 
 
-### Warm Restart show commands
+### Advanced Restart show commands
 
-**show warm_restart config**
+**show advanced_restart config**
```

Comment:
I don't think we can rename CLI. From user's point of view there are two different kinds of reboot - fast and warm and the implementation is mostly shared. We should not rename CLI.

Reply:
Thanks for the note, I agree with you and reverted all changes to file. I wrote to Shlomi, trying to understand his intent - IMHO advanced boot is an abstraction that is done in the backstage and user should not be aware of it. He can use either warm/fast/fastfast and doesn't care about advanced.